lib/instances/init.lua

```diff
@@ -13,6 +13,7 @@ local names = {
 	"ScreenGui",
 	"Script",
 	"StringValue",
+	"Workspace",
```

Comment:
We should also add `Camera` here.

Don't forget to make `Game` instantiate a Workspace so that `game:GetService("Workspace")` and `game.Workspace` both work!

Reply:
Workspace is a property, meaning if you change the name of Workspace you can still do `game.Workspace`.

Reply:
I think it's safe to disallow the renaming of services in Lemur to make that a non-issue @_@